internal/monitor/exporter.go

```diff
@@ -25,20 +24,72 @@ import (
 	"contrib.go.opencensus.io/exporter/ocagent"
 	"contrib.go.opencensus.io/exporter/prometheus"
 	"contrib.go.opencensus.io/exporter/stackdriver"
+	"github.com/googlecloudplatform/gcsfuse/v2/cfg"
+	"github.com/googlecloudplatform/gcsfuse/v2/common"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/logger"
 	"go.opencensus.io/stats/view"
 )
 
-var stackdriverExporter *stackdriver.Exporter
+func maybeStartStackdriverExporter(exporterIntervalSecs int64) common.ShutdownFn {
```

Comment:
nit: mayBe in the method name sounds little different to me. I have this for library functions but in our case, its just about not failing the mount flow if the exporter is not started. Can we keep it simple startStackDriverExporter.